### DISCUSSION TITLE
Please implement the frickin' offset wizard.

### DISCUSSION THREAD

**Discussion by ItsRedly** [ppy/osu#24908] [Ideas]
Hey, osu!mania player here. I can't figure out my perfect offset since the wizard isn't here. The only temporary workaround is installing osu!stable and doing it there, then going back to osu!lazer and setting the global offset to that

**Comment by bdach** [comment]
[please frickin' search](https://github.com/ppy/osu/issues/9866)

**Reply by ItsRedly** [reply]
That's from 2020. Plus, it doesn't seem like that thread'll go anywhere-

(i found it before)

**Reply by longnguyen2004** [reply]
From 2020 doesn't mean that it's ignored by the maintainers. Also they have [143 priority 1 issues](https://github.com/ppy/osu/issues?q=is%3Aissue+is%3Aopen+label%3Apriority%3A1) open, and also the ranked play project to work through. Give them time.

**Comment by peppy** [comment]
As a temporary workaround, you can download the offset wizard beatmap, hit ctrl-o and adjust the offset until it feels correct.

Or if you prefer your offset to be based on your click timing, play a beatmap once then retry and take note of the autoamtic calibration result, and set that on your global offset.
